examples/STM32L475VG-DISCOVERY-IOT/BTLE_sensors_tof_demo/BTLE_sensors_tof_demo.ino

```diff
@@ -0,0 +1,422 @@
+/*
+
+ BTLE_sensors_tof_demo
+
+ This sketch provides a default demo to be used on STMicroelectronics Discovery L475VG IoT board.
+ It will use several components of the board :
+   * Low energy Bluetooth (SPBTLE_RF)
+   * temperature and pressure sensor (LPS22HB)
+   * temperature and humidity sensor (HTS221)
+   * Time Of Flight sensor (VL53L0X)
+
+ This sketch will launch 3 services on BLE : Acc, Environnemental and Time.
+ For testing the sketch, you can download on the playstore the "BlueNRG" application provided by
+ STMicroelectronics.
+ Enable Bluetooth and launch the application on your smartphone. Connect it to the BLueNRG device.
+ You will see all the services, you can click on each one.
+
+ Temperature, humidity and pressure are sent to the environment service and updated on regular
+ basis.
+ To move the cube on motion tab, you have to swipe on top of the vl53l0x sensor. To change
+ direction of the motion, use a tap. The gesture has to be done at less than 1 meter from the
+ sensor.
+
+ Do not forget to check information on the console.
+
+ */
+
+#include <SPI.h>
+#include <SPBTLE_RF.h>
+#include <sensor_service.h>
+
+#include <HTS221Sensor.h>
+#include <LPS22HBSensor.h>
+
+#include <vl53l0x_class.h>
+#include <tof_gestures.h>
+#include <tof_gestures_SWIPE_1.h>
+#include <tof_gestures_TAP_1.h>
+
+/* Configure SPI3 for BLE
+  MOSI: PC12 (44), MISO: PC11 (43), SCLK: PC10 (42)
+  */
+SPIClass SPI_3(44, 43, 42);
```

Comment:
Please, use PXn instead of pin numbering when higher than 15 (arduino header D0-D15).